app/views/datasets/licences/edit.html.erb

```diff
@@ -23,32 +23,7 @@
   <h1 class="heading-large">Choose a licence for this dataset</h1>
 
   <%= form_for @dataset, url: update_dataset_licence_path(@dataset.uuid, @dataset.name) do |f| %>
-    <div class="form-group">
-      <fieldset>
-        <legend class="visually-hidden">Choose a licence for this dataset</legend>
-
-        <%= dataset_field f, @dataset,
-        name: 'licence',
-        input_type: :radio_button,
-        label: 'Open Government Licence',
-        value: 'uk-ogl' %>
-
-        <%= dataset_field f, @dataset,
-        target: 'other-licence',
-        name: 'licence',
-        input_type: :radio_button,
-        label: 'Other',
-        value: 'other' %>
-
-        <div class="panel panel-border-narrow js-hidden" id="other-licence">
-          <%= dataset_field f, @dataset,
-          input_type: :text_field,
-          name: 'licence_other',
-          label: 'Name of your licence:' %>
-        </div>
-      </fieldset>
-    </div>
-
+    <%= render partial: 'licence_options', :locals => {f: f} %>
```

Comment:
It's good practice to _tell_ the partial everything it needs to know to render to make it more reusable.

We should tell the partial about the dataset so that it doesn't depend on `@dataset` being available in it's rendering context.

```rb
<%= render partial: 'licence_options', locals: { f: f, dataset: @dataset } %>
```